dotcom-rendering/src/web/components/SignedInAs.tsx

```diff
@@ -83,7 +83,7 @@ const rowUntilDesktop = css`
 	}
 `;
 
-const CommentCount = ({ count }: { count?: number }) => {
+const Heading = ({ count }: { count?: number }) => {
```

Comment:
This was just a drive by refactor because this unrelated component with the same name kept coming up in my searches